docs/source/system_reference_guide/s3-inventory.ipynb

```diff
@@ -0,0 +1,2025 @@
+{
```

Comment:
<p>Number of objects under a collection prefix might be more telling than total size in bytes</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/220/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>If we expect objects to have similar sizes across collections yes. Is it the case ? </p><p> </p><p>Otherwise it doesn't seem the number of objects would be very informative, if our goal is to rank the collections in terms of storage space used. I might be misunderstanding the goal, though :)</p><p> </p>